README.md

```diff
@@ -154,6 +154,41 @@ Composite index keys are not supported.
 
 The primary key isn't indexed by default.
 
+## Serialized fields
+
+The `attribute` method can be used to provide a custom class to deserialize an attribute.  The class must implement a
+`load` class method that takes the raw attribute value and returns the deserialized value (similar to
+[ActiveRecord serialization](https://api.rubyonrails.org/v7.0.4/classes/ActiveRecord/AttributeMethods/Serialization/ClassMethods.html#method-i-serialize)).
+
+```ruby
+class ContinentString < String
+  class << self
+    alias_method :load, :new
+  end
+end
+
+class CurrencyCodeEnum
+  CAD = "CAD"
+  EUR = "EUR"
+  class << self
+    def load(value)
+      if value == "CAD"
+          CAD
+      elsif value == "EUR"
+          EUR
```

Comment:
Nitpick:, this example is a bit weird, because you could just use `String#-@` to the same effect.

Do you have any better example? I don't know some kind of `Struct.new`? Or even a `Money` class?